### DISCUSSION TITLE
Is Karpenter Module Supported in Gruntworks Currently ?

### DISCUSSION THREAD

**Discussion by vramanufidato** [gruntwork-io/knowledge-base#634] [How do I] [answered]
Hi Team,

As per our understanding the Karpenter module is currently not supported ? [Karpenter module](https://github.com/gruntwork-io/knowledge-base/discussions/563) So it needs to be developed as a completely new module as mentioned create totally new services ? [New Service](https://docs.gruntwork.io/reference/services/intro/create-your-own-service-catalog#creating-a-service-catalog)

---

<ins datetime="2023-01-17T08:56:01Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109796">Tracked in ticket #109796</a></p>
</ins>

**Comment by josh-padnick** [comment, marked as answer]
This question is best tracked in the underlying GitHub Issue at https://github.com/gruntwork-io/terraform-aws-eks/issues/379.
